### DISCUSSION TITLE
Separate VLAN for OS and Mesh Commander

### DISCUSSION THREAD

**Discussion by itayiad** [Ylianst/MeshCentral#6334] [General]
Is it possible to have mesh commander being assigned a different IP address to the host OS

**Comment by si458** [comment]
I'm not sure I follow?
Meshcommander is an application you use to connect to intel amt devices?
What do u mean u want different ip addresses?
Do u mean u want the intel amt to have a different ip address than whatever the OS ip address is?
That's possible but u mist assign a static ip address inside the intel amt mbex, u can't give amt its own dhcp ip address as it shares the mac address from ur OS
